cfme/tests/v2v/test_migration_throttling.py

```diff
@@ -0,0 +1,91 @@
+import fauxfactory
+import pytest
+
+from widgetastic.widget import NoSuchElementException
+
+from cfme import test_requirements
+from cfme.cloud.provider.openstack import OpenStackProvider
+from cfme.fixtures.provider import rhel69_template
+from cfme.fixtures.provider import rhel7_minimal
+from cfme.fixtures.provider import ubuntu16_template
+from cfme.infrastructure.provider.rhevm import RHEVMProvider
+from cfme.infrastructure.provider.virtualcenter import VMwareProvider
+from cfme.markers.env_markers.provider import ONE_PER_TYPE
+from cfme.markers.env_markers.provider import ONE_PER_VERSION
+
+pytestmark = [
+    test_requirements.v2v,
+    pytest.mark.provider(
+        classes=[OpenStackProvider, RHEVMProvider],
+        selector=ONE_PER_VERSION,
+        required_flags=["v2v"],
+        scope="module"
+    ),
+    pytest.mark.provider(
+        classes=[VMwareProvider],
+        selector=ONE_PER_TYPE,
+        fixture_name="source_provider",
+        required_flags=["v2v"],
+        scope="module",
+    ),
+    pytest.mark.usefixtures("v2v_provider_setup"),
+]
+
+
+@pytest.mark.parametrize(
+    "mapping_data_multiple_vm_obj_single_datastore",
+    [
+        ["nfs", "nfs", [rhel7_minimal, rhel69_template, ubuntu16_template]],
+    ],
+    indirect=True,
+)
+def test_migration_throttling(
+    request, appliance, mapping_data_multiple_vm_obj_single_datastore
+):
+    """
+    Polarion:
+        assignee: sshveta
+        casecomponent: V2V
+        initialEstimate: 1/4h
+        caseimportance: high
+        caseposneg: positive
+        testtype: functional
+        startsin: 5.10
+    """
+    migration_throttling_collection = appliance.collections.v2v_throttling_settings
+    # Setting throttling at 2 concurrent migration per conversion host
+    migration_throttling_collection.set_max_migration_per_conv_host("2")
+
+    infrastructure_mapping_collection = appliance.collections.v2v_infra_mappings
+    mapping_data = mapping_data_multiple_vm_obj_single_datastore.infra_mapping_data
+    mapping = infrastructure_mapping_collection.create(**mapping_data)
```

Comment:
Can this `mapping` be moved into a fixture? Since the test is on throttling and not creating the mapping.

Reply:
@john-dupuy : Infrastructure mapping takes mapping data which is different for each test and is passed from test parameters . Not able to create a fixture for it